<launch>
    <!-- <param name="use_sim_time" value="true" /> -->

    <arg name="PATH_TO_VOCABULARY" value="$(find rds_slam)/Vocabulary/ORBvoc.txt" />
    <arg name="PATH_TO_SETTINGS_FILE" value="$(find rds_slam)/config/RGB-D/TUM3.yaml" />
    <arg name="PATH_TO_SEQUENCE" value="/root/Dataset/TUM/freiburg3/rgbd_dataset_freiburg3_walking_static" />
    <arg name="PATH_TO_ASSOCIATIONS" value="$(find rds_slam)/config/RGB-D/associations/fr3_walk_static.txt" />

    <node name="rds_slam_tum" pkg="rds_slam" type="Dynamic_RGBD" args="$(arg PATH_TO_VOCABULARY) $(arg PATH_TO_SETTINGS_FILE) $(arg PATH_TO_SEQUENCE) $(arg PATH_TO_ASSOCIATIONS)" output="screen" >
        <param name="cnn_method" type="string" value="maskrcnn" />
        <param name="init_delay" type="double" value="30000" />
        <param name="frame_delay" type="double" value="30000" />
        <param name="init_frames" type="int" value="1" />
    </node>

</launch>
